js/components/form/zone-completer.vue

```diff
@@ -120,31 +119,33 @@ export default {
                     opt.push('</div>');
                     return opt.join('');
```

Comment:
In order to improve readabilty and because you are rewriting into ES6, I would use the ES6 template syntax:

```js
const opt = `
    <div class="selectize-zone">
        <span class="title">
            <span class="name">${escape(data.name)}</span>
            <span class="code">${escape(data.code)}</span>
        </span>
        <span class="level">${this.levelLabel(data)}</span>`;
if (data.keys) {
    opt += '<ul>';
    Object.keys(data.keys).map(key => {
        opt += `
            <li>
                <span class="text-uppercase">${opt.push(escape(key))}</span>: 
                <span class="value">${escape(data.keys[key])}</span>
            </li>`;
    opt += '</ul>';
}
opt += '</div>';
return opt;
```